.gitignore

```diff
@@ -8,3 +8,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# IDE settings
+.vscode/*
```

Comment:
blasphemy!

Reply:
So what, I use [rust-analyzer](https://github.com/rust-analyzer/rust-analyzer), sue me! :joy:

Reply:
You can also use rust-analyzer in neovim :wink: